src/handlers/gestureHandlerCommon.ts

```diff
@@ -111,6 +111,8 @@ export type ActiveCursor =
   | 'zoom-in'
   | 'zoom-out';
 
+export type EnableContextMenu = boolean;
+
```

Comment:
This seems unrelated, no?

Reply:
Done in[9936d77] (https://github.com/software-mansion/react-native-gesture-handler/pull/2680/commits/9936d772b4ff4a5e1183c57f8377863d9a3874d9)

Reply:
Done in [9936d77](https://github.com/software-mansion/react-native-gesture-handler/pull/2680/commits/9936d772b4ff4a5e1183c57f8377863d9a3874d9)